sdk/servicebus/service-bus/test/README.md

```diff
@@ -2,7 +2,9 @@
 
 - Run `npm i` to install all the dependencies of this project. This is a one time task.
 - The tests expect a series of queues, topics and subscriptions to exist in a single Service Bus instance.
-  The connection string for the service bus should be in the environment variable `SERVICEBUS_CONNECTION_STRING`.
+  For Node environment, the connection string for the service bus should be in the environment variable `SERVICEBUS_CONNECTION_STRING`.
+
+  For browser environment, the connection string for the service bus should be in the environment variable `SERVICEBUS_CONNECTION_STRING_BROWSER`.
 
   Note that the tests will empty the messages in these entities to get a clean start before running each test.
```

Comment:
After merging the `feature/service-bus-atom` branch, all tests now create the entities they need. So the above statement is no longer true. So this note and the table following it can be removed.

Similarly the sample content for the .env file should be cleaned up as well